js/README.md

```diff
@@ -303,86 +303,84 @@ This project provides an ONNX Runtime React Native JavaScript library to run ONN
-   ```
+    1. To use a published Android ONNX Runtime Mobile package from Maven, go to step 5.
 
-   3. This generates `onnxruntime-mobile-<version>.aar` in `<BUILD_DIRECTORY>/aar_out/MinSizeRel/com/microsoft/onnxruntime/onnxruntime-mobile/<version>`. Copy `aar` file into `<ORT_ROOT>/js/react_native/android/libs` and rename it as `onnxruntime.aar`
+    2. Set up an Android build environment referring to [instruction](https://www.onnxruntime.ai/docs/how-to/build/android-ios.html#android)
 
-   4. To verify, open Android Emulator and run this command from `<ORT_ROOT>/js/react_native/android`
+    3. In `<ORT_ROOT>`, run this python script to build ONNX Runtime Android archive file. In windows, this requires an admin account to build. To build a model specific package with reduced size, refer to [instruction](https://www.onnxruntime.ai/docs/how-to/build/reduced.html#build-ort-with-reduced-size).
 
-   ```sh
-   adb shell am instrument -w ai.onnxruntime.react_native.test/androidx.test.runner.AndroidJUnitRunner
-   ```
+        ```sh
+        python tools/ci_build/github/android/build_aar_package.py tools/ci_build/github/android/default_mobile_aar_build_settings.json --config MinSizeRel --android_sdk_path <ANDROID_SDK_PATH> --android_ndk_path <ANDROID_NDK_PATH> --build_dir <BUILD_DIRECTORY> --include_ops_by_config tools/ci_build/github/android/mobile_package.required_operators.config
+        ```
 
-3. Build iOS ONNX Runtime package
+    4. Copy `<BUILD_DIRECTORY>/aar_out/MinSizeRel/com/microsoft/onnxruntime/onnxruntime-mobile/<version>/onnxruntime-mobile-<version>.aar` into `<ORT_ROOT>/js/react_native/android/libs` directory.
 
-   1. Set up iOS build environment referring to [instruction](https://www.onnxruntime.ai/docs/how-to/build.html#ios).
+    5. Modify `Onnxruntime_mobileVersion` property in `<ORT_ROOT>/js/react_native/android/build.properties` to consume a locally built package or a newly published package from Maven.
 
-   2. Build ONNX Runtime library for iOS from `<ORT_ROOT>` using this command,
+    6. To verify, open Android Emulator and run this command from `<ORT_ROOT>/js/react_native/android`
 
-   ```sh
-   ./build.sh --config MinSizeRel --use_xcode --ios --ios_sysroot iphoneos --osx_arch arm64 --apple_deploy_target 11
-   ```
+        ```sh
+        adb shell am instrument -w ai.onnxruntime.react_native.test/androidx.test.runner.AndroidJUnitRunner
+        ```
 
-   Copy `<ORT_ROOT>/build/iOS/MinSizeRel/MinSizeRel-iphoneos/libonnxruntime.<version>.dylib` file into `<ORT_ROOT>/js/react_native/ios/Libraries/onnxruntime/lib/iphoneos`
+3. Build iOS ONNX Runtime package
 
-   3. Clean up the previous build and build ONNX Runtime library for iOS Simulator from `<ORT_ROOT>`
+    1. Set up iOS build environment referring to [instruction](https://www.onnxruntime.ai/docs/how-to/build/android-ios.html#ios).
 
```

Comment:
We can use the iOS c/c++ package on CocoaPods also 
[https://www.onnxruntime.ai/docs/how-to/mobile/initial-setup.html#cc-1](https://www.onnxruntime.ai/docs/how-to/mobile/initial-setup.html#cc-1)

Reply:
I simply tested c/c++ CocoaPods, it works perfectly. Now, we don't need to add both Android and iOS ONNX Runtime into ONNX Runtime React Native npm package, so npm package size will be only 21KB.